src/kirby/components/list/list.component.html

```diff
@@ -14,38 +16,19 @@
       </ion-item-divider>
 
       <div class="list-items">
-        <ion-item *ngFor="let item of section.items; let idx = index;trackBy: trackByFn"
-                  [class.has-divider]="showDivider"
-                  [class.selectable]="isSelectable"
-                  [class.selected]="markSelectedRow && selectedItem && item._id === selectedItem._id"
-                  [attr.role]="isSelectable ? 'button' : null"
-                  [class.last-in-section]="idx === section.items.length - 1"
-                  (click)="onItemSelect(item)"
-                  tabindex="0"
-                  [kirbyListItemColor]="getItemColor"
-                  [item]="item"
-                  keyHandler>
-          <ng-container *ngTemplateOutlet="listItemTemplate; context: { $implicit: item }"></ng-container>
-          <ng-container *ngTemplateOutlet="listFlexItemTemplate; context: { $implicit: item }"></ng-container>
-        </ion-item>
+        <ion-item-sliding *ngFor="let item of section.items; let idx = index;trackBy: trackByFn" [disabled]="isSwipingDisabled">
```

Comment:
I think now that we've moved the item html into it's own template we should do the same regarding `<ion-item-sliding>` wrapper / plain items:
```suggestion
          <ng-container *ngTemplateOutlet="hasSwipeActions ? itemSlidingTemplate : itemTemplate; context: { $implicit: item }"></ng-container>
```